packages/modelence/src/email/loadProvider.ts

```diff
@@ -0,0 +1,45 @@
+import { EmailProvider, EmailProviderName } from './types';
+
+export async function loadProvider(name: EmailProviderName): Promise<EmailProvider> {
```

Comment:
Dynamically importing these based on an input string seems problematic, because we would have to include every single email provider in the bundle. Instead of doing this can we just create a separate npm package wrapper for each provider, that exports its own `sendEmail` method? We can define a unified interface that each of them have to implement, but they don't even have to exactly match in case some providers have core differences.

I remember we discussed this earlier and decided to go with that separate package approach to avoid bundling providers with the core framework.